src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Location/index.js

```diff
@@ -9,51 +10,55 @@ import MapDetail from '../../../MapDetail';
 
 import './style.scss';
 
-const Location = ({
-  incident, stadsdeelList, onShowLocation, onEditLocation,
-}) => {
+const MapButton = styled.button`
+  float: left;
+  margin-right: 10px;
+  padding: 0;
+  border-style: none;
+  cursor: pointer;
+`;
+
+const Location = ({ incident, stadsdeelList, onShowLocation, onEditLocation }) => {
   const address = incident.location.address;
   return (
     <dl className="location">
-      <dt className="location__definition" data-testid="location-definition">Locatie</dt>
+      <dt className="location__definition" data-testid="location-definition">
+        Locatie
+      </dt>
       <dd className="location__value">
         <button
           className="location__value-button-edit incident-detail__button--edit"
           type="button"
           onClick={onEditLocation}
           data-testid="location-button-edit"
         ></button>
-        <button
-          className="location__value-button-show"
-          type="button"
-          onClick={onShowLocation}
-          data-testid="location-button-show"
-        >
+        <MapButton type="button" onClick={onShowLocation} data-testid="location-button-show">
           <MapDetail
             value={incident.location}
-            hideAttribution
-            hideZoomControls
-            useSmallMarker
-            zoom="15"
+            smallMarker
+            attributionControl={false}
+            zoom={15}
+            width="80px"
```

Comment:
Isn't it possible to set height and width of `MapDetail` by making it into a styled component?
Like so:
```jsx
const StyledMap = styled(MapDetail)`
  height: 80px;
  width: 80px;
`;
```

Reply:
done